common/data/packages_micro.json

```diff
@@ -1,15 +1,13 @@
 {
-    "packages":["glibc","zlib", "filesystem",
-                "gmp", "mpfr", "libgcc", "libstdc++",
-                "bzip2", "pkg-config", "ncurses", "cracklib", "cracklib-dicts", "shadow", "procps-ng", "e2fsprogs",
-                "iana-etc", "readline", "coreutils", "bash",
-                "inetutils",
-                "xz", "iproute2", "kbd", "kmod", "libpipeline",
+    "packages":[
```

Comment:
Only the packages that should be in Micro and Minimal must be added to the json files. The dependent packages will be automatically packaged & installed.